src/rpc/rawtransaction.cpp

```diff
@@ -905,7 +905,7 @@ static RPCHelpMan testmempoolaccept()
                 RPCResult{
                     RPCResult::Type::ARR, "", "The result of the mempool acceptance test for each raw transaction in the input array.\n"
                         "Returns results for each transaction in the same order they were passed in.\n"
-                        "It is possible for transactions to not be fully validated ('allowed' unset) if an earlier transaction failed.\n",
+                        "It is possible for transactions to not be fully validated ('allowed' unset) if another transaction failed.\n",
```

Comment:
In [refactor] comment/naming improvements

I think this is a bit confusing: the reader hasn't be introduced to the `allowed` field yet, so they don't know it's part of the results.  When I read this, I thought it was an input parameter that I had missed.  I don't think this is important, but if you edit, maybe this would be clearer:

    "Transactions that cannot be fully validated due to failures in other transactions will not contain an 'allowed' result.\n"

Reply:
Good point, I'll do this if I need to retouch or in a followup

Reply:
Added to #21800